File: Python/Manual01.py/Question09.py
gpa = float(input("Enter your GPA: "))

if gpa < 0.0 or gpa > 4.0:
    print("Enter a Valid GPA")
elif gpa <= 0.99:
    print("Failed Semester")
elif gpa <= 1.99:
    print("On Probation for next Semseter")
elif gpa <= 2.99:
    print("No Comments")
elif gpa <= 3.49:
    print("Dean's List for Semester")
elif gpa <= 4.0:
    print("Highest Honors for Semester")

File: JavaScript/Manual 01/Question09.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter your GPA: ", function (gpa) {
    if (gpa <= 0.99)
        console.log("Failed Semester");
    else if (gpa <= 1.99)
        console.log("On Probation for next Semester");
    else if (gpa <= 2.99)
        console.log("No Comments");
    else if (gpa <= 3.49)
        console.log("Dean's List for Semester");
    else if (gpa <= 4.00)
        console.log("Highest Honors for Semester");
    else
        console.log("Invalid Input");

    rl.close();
});